packages/api/src/api/project/post.ts

```diff
@@ -0,0 +1,36 @@
+import { Schema } from '@hangar/shared';
+import { Request, Response } from 'express';
+import { logger } from '../../utils/logger';
+import { validatePayload } from '../../utils/validatePayload';
+import { Project, User } from '@hangar/database';
+
+export const post = async (req: Request, res: Response) => {
+  const { entityManager: em, user } = req;
+
+  const { errorHandled, data } = validatePayload({ req, res, schema: Schema.project.post });
+  if (errorHandled) return;
+  
+  const query = {id:req.user.id};
+  
+  // check for existing project
+  try {
+    const user = await em.findOne(User, query);
+    if (user?.project!==undefined) {
+      res.status(409);
+      return;
+    }
```

Comment:
Honestly we don't even need to make the query... if the `user` has a `project` at ALL, it means the entity exists and you should throw a 409...
```suggestion
    if (user?.project) {
      res.status(409);
      return;
    }
```